src/App.tsx

```diff
@@ -1,52 +1,79 @@
+import {
+  NavLink, Route, Navigate, Routes,
+} from 'react-router-dom';
+import classNames from 'classnames';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
+import { TabsPage } from './component/Tabs';
 
-// const tabs = [
-//   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
-//   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
-//   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
-// ];
+export const App = () => {
+  return (
+    <>
+      <nav
+        className="navbar is-light is-fixed-top is-mobile has-shadow"
+        data-cy="Nav"
+      >
+        <div className="container">
+          <div className="navbar-brand">
+            <NavLink
+              className={({ isActive }) => (
+                classNames(
+                  'navbar-item',
+                  {
+                    'is-active': isActive,
+                  },
+                )
+              )}
```

Comment:
feels weird
`isActive` should be a variable, right?
where do you declare and initialise it?
why do you need a whole callback?

same for the second NavLink
```suggestion
              className={classNames(
                'navbar-item',
                {
                  'is-active': isActive,
                },
              )}
```